images/base/Dockerfile

```diff
@@ -79,6 +79,9 @@ RUN echo "Ensuring scripts are executable ..." \
     && rm -f /lib/systemd/system/basic.target.wants/* \
     && echo "ReadKMsg=no" >> /etc/systemd/journald.conf \
     && ln -s "$(which systemd)" /sbin/init \
+ && echo "Using iptables-nft ..." \
```

Comment:
should we not do this in the job level?
the docker network stuff injects rules to the kind "node"